pkg/envoy/eds/response.go

```diff
@@ -56,6 +53,7 @@ func fulfillEDSRequest(meshCatalog catalog.MeshCataloger, proxy *envoy.Proxy, re
 			log.Error().Err(err).Msgf("Failed listing allowed endpoints for service %s, for proxy identity %s", meshSvc, proxyIdentity)
 			continue
 		}
+		log.Debug().Msgf("FOO Endpoints for upstream service %s for downstream proxy identity %s: %v", cluster, proxyIdentity, endpoints)
```

Comment:
Is this a remnant from a debugging session?

Reply:
Yes, removed the `FOO` string and made this log a `trace` to aid debugging.